data/blog/fonts.mdx

```diff
@@ -116,16 +116,18 @@ Font files contain multiple languages and glyphs, which increase the file size.
 
 For example, we might use the Inter variable font and subset to latin languages.
 
+Several tools are available to subset fonts - such as:
+- [Font Squirrel](https://www.fontsquirrel.com/tools/webfont-generator) (web based)
+- [Fontie](https://fontie.pixelsvsbytes.com/webfont-generator) (web based)
+- [Fontforge](https://fontforge.org/) (open source Mac/Windows/Linux tool)
+
 ```css
 @font-face {
   font-family: 'Inter';
   font-style: normal;
   font-weight: 100 900; // Range of weights supported
   font-display: optional;
   src: url(/fonts/inter-var-latin.woff2) format('woff2');
-  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
```

Comment:
I believe this unicode range is still correct for Inter - or no?

Reply:
I guess I'm not sure if that unicode range specifically is correct for the Inter font (for basic latin characters?). I'm not sure it serves the purpose the article makes because it isn't only downloading the characters specified by unicode-range, but instead downloading the entire font if any of the characters specified in unicode-range are found.